ci/requirements-ac-test.txt

```diff
@@ -24,3 +24,4 @@ tqdm==4.42.1
 wcwidth==0.1.8            # via pytest
 yamlloader==0.5.5
 zipp==1.1.0               # via importlib-metadata
+editdistance==0.5.3
```

Comment:
Please use the `update-requirements.py` script to update this file.

Reply:
Hi, thanks for your careful review. I'm not sure about how to use the script to update this file. I've run this script. It didn't seem to do anything... Would you please explain a little bit more? Thanks!

Reply:
Did you get any output from the script?

Reply:
(base) upsquared@UP2:~/open_model_zoo/ci$ ~/anaconda3/envs/py35_env/bin/python update-requirements.py
updating ci/requirements-ac.txt...
/home/upsquared/anaconda3/envs/py35_env/bin/python: No module named piptools
Traceback (most recent call last):
  File "update-requirements.py", line 58, in <module>
    main()
  File "update-requirements.py", line 46, in main
    'tools/accuracy_checker/requirements.in')
  File "update-requirements.py", line 34, in pip_compile
    check=True, cwd=str(repo_root))
  File "/home/upsquared/anaconda3/envs/py35_env/lib/python3.5/subprocess.py", line 398, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/home/upsquared/anaconda3/envs/py35_env/bin/python', '-mpiptools', 'compile', '--no-header', '--upgrade', '--quiet', '-o', 'ci/requirements-ac.txt', '--', 'tools/accuracy_checker/requirements.in']' returned non-zero exit status 1

Reply:
You need to install the `pip-tools` package, preferably version 4.4.1.

Reply:
Hi, after installing the pip-tools==4.4.1. The script can run successfully. However, after the running, not only the editdistance has been added, some other packages has been changed and updated. Like this:
-nibabel==3.0.1
-numpy==1.17.5             # via nibabel, scikit-learn, scipy
-packaging==20.1           # via pytest
+nibabel==3.0.2
+numpy==1.18.1             # via nibabel, scikit-learn, scipy
+packaging==20.3           # via pytest
Are such modifications expected? Would you please tell me what should I do about it? Thanks!

Reply:
If such modifications are not allowed, could you please help me run the script `update-requirements.py` on your side? Because in my anaconda python3.5 environment, such extra modifications will always happen when I run this script... Many thanks!

Reply:
Also, there's always an HTTPSConnectionPool due to the network problem...

Reply:
> Are such modifications expected? Would you please tell me what should I do about it? Thanks!

It should be fine, although if you want to stay on the safe side, you can revert the changes, temporarily edit the script to remove the string `'--upgrade'` and rerun it. This will prevent unrelated packages from being upgraded.